api/bundles.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/ONSdigital/dis-bundle-api/application"
+	dpresponse "github.com/ONSdigital/dp-net/v3/handlers/response"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (api *BundleAPI) getBundles(w http.ResponseWriter, r *http.Request, limit, offset int) (interface{}, int, error) {
+	ctx := r.Context()
+
+	bundles, totalCount, err := application.Bundlestore.ListBundles(api.bundleAPI.Store, ctx, offset, limit)
+	if err != nil {
+		log.Error(ctx, "failed to get bundles", err)
+		http.Error(w, "internal server error", http.StatusInternalServerError)
```

Comment:
These errors should be returned with the response body in the Error and Error list models, I don't see any reference to those being returned.

Also, is this definitely an internal server error?  Could it be a 404 or a 401 instead which are not internal server errors?